activities/codebase-auditing/review-template.md

```diff
@@ -0,0 +1,187 @@
+---
+type: Resource
+---
+
+# [Codebase] and the Standard for Public Code
+
+## [Code in the Open](https://standard.publiccode.net/criteria/code-in-the-open.html)
+
+- [ ]
```

Comment:
@Ainali what are these for?

Reply:
Is this a tickbox so that the whole criterion can be ticked off when it's complete?

Reply:
Maybe, however since there is no content it is not recognised as a tickbox, since that needs something to tick.

Perhaps then it needs to be:

```suggestion
- [ ] compliant with this criterion.
```

Reply:
It's a tick box for the whole criterion. You can see it in use here: https://github.com/open-zaak/open-zaak/blob/master/docs/public_code.md

Reply:
I like it, however, it is without content not a task list item ([spec](https://github.github.com/gfm/#task-list-items-extension-)) so perhaps we can add some content that describes what to put there. Which we can do in square brackets `[summary of your findings]`.

What do you think of my above suggestion? Can I add it to all of them?

Reply:
Or perhaps this, lightweight:

```suggestion
- [ ] fulfilled.
```

@Ainali, would you be okay with this for now?

Reply:
Sounds like a good idea! Personally, I like your first suggestion better than the lightweight one.